cmake/CubCudaConfig.cmake

```diff
@@ -11,8 +11,11 @@ set(arch_message "CUB: Explicitly enabled compute architectures:")
 # reuse them if possible. After we transition to CMake 3.18 CUDA_ARCHITECTURE
 # target properties this will need to be updated.
 if (CUB_IN_THRUST)
-  # Configure to use all flags from thrust:
-  set(CMAKE_CUDA_FLAGS "${THRUST_CUDA_FLAGS_BASE} ${THRUST_CUDA_FLAGS_NO_RDC}")
+  # Configure to use all flags from thrust. See ThrustCudaConfig.cmake for
+  # details.
+  set(CUB_CUDA_FLAGS_BASE "${THRUST_CUDA_FLAGS_BASE}")
```

Comment:
Nitpick: I do not like `BASE`, would it be possible to change that to `CUB_CUDA_FLAGS_COMMON`

Reply:
Good news :) We'll eventually rip all of these out when we are able to migrate to using the `CUDA_ARCHITECTURE` target property. I'll leave these as-is since these variables are just temporary.